helx/charts/search/values.yaml

```diff
@@ -138,7 +140,7 @@ airflow:
       enabled: true
       repo: "https://github.com/helxplatform/roger.git"
       repoSubPath: "dags"
-      branch: "master"
+      branch: "develop"
```

Comment:
I wonder if we should leave this as master, since that is the "released" version of the dags. It can always be overriden on a per-environment basis though

Reply:
makes sense, Changed back to master (was just holding off till we have airflow 2.0 compatible release for roger).